trymito.io/pages/teams.tsx

```diff
@@ -0,0 +1,232 @@
+                        </p>
+                        <a href="mailto:founders@sagacollab.com" className={pageStyles.link_with_p_tag_margins}>
+                            Contact the Mito team →
+                        </a>
+                    </div>
+                    <div className={homeStyles.functionality_media}>
+                        <Image src={'/explore.png'} alt='Explore your data with Mito' width={500} height={250} layout='responsive'/>
+                    </div>
+                </div>
+            </section>
+            <section>
+                <div className={pageStyles.subsection + ' center'}>
+                    <h1>
+                        Deploy Mito in Jupyter in minutes
+                    </h1>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={JupyterIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Jupyter <br/> Compatible
+                        </h1>
+                        <p>
+                            Mito is compatible with Jupyter Hub, Jupyter Lab, and Jupyter Notebooks, so you don’t need to manage any new infrastructure.  
+                        </p>
+                    </div>
+                    <div className={securityStyles.security_bullet_container + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={PrivateIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Completely <br/> Private
+                        </h1>
+                        <p>
+                            With the Mito Pro and Enterprise plans, not a single message is sent back to Mito. Mito is designed from the ground up with security at its core. 
+                        </p>
+                    </div>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={DocsIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Practical <br/> Documentation
+                        </h1>
+                        <p>
+                            Great documentation is a must even for the most intutive tools. Mito’s docs are easily consumable, so your users can get started without any training.  
```

Comment:
> Mito’s docs are easily consumable, so your users can get started without any training.

- Our documentation is best-in-class, making getting started a breeze.
- Our documentation is best-in-class, so you can get started without complex trainings.